bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -45,197 +41,80 @@
-{% endif %}
-</p>
-
-{% endblock %}
-
-{# The contents of the dashboard_faq block appears below the intro. For each question/answer
-   use the following format:
-
-    <section>
-      <h3 data-accordion-role="tab">{{ _('Question title goes here?') }}</h3>
-      <div data-accordion-role="tabpanel">
-        <p>
-          {% trans %}
-            Question answer goes here.
-          {% endtrans %}
-        </p>
-      </div>
-    </section>
-
-#}
-
-{% block dashboard_faq %}
-<section id="faq" class="accordion accordion-auto-init zebra">
-
-<h2>{{ _('Frequently Asked Questions') }}</h2>
-
-<section>
-  <h3 data-accordion-role="tab">{{ _('Are you tracking users now?') }}</h3>
-  <div data-accordion-role="tabpanel">
-    <p>
-      {% trans %}
-        We only collect data from the New Tab page in aggregate form, meaning no data is personally identifiable.
-      {% endtrans %}
-    </p>
-  </div>
+<section class="tls-personalize">
+  <h2>{{ _('Personalize your New Tab experience') }}</h2>
+  <p>
+    {{ _('You are in complete control of your New Tab page.') }}
+    {{_('Customize your experience by rearranging, pinning or removing sites, and set up New Tab just the way you want it.') }}
+  </p>
+  <ol>
+    <li>{{ _('Drag a site to exactly where you want it.') }}
+      <img src="{{ static('img/firefox/tiles/drag-site.png') }}" alt="{{ _('Screenshot') }}">
+    </li>
+    <li>{{ _('Once you’ve found the perfect spot for your site, pin it to make it stay.') }}
+      <img src="{{ static('img/firefox/tiles/pin-site.png') }}" alt="{{ _('Screenshot') }}">
+    </li>
+    <li>{{ _('Make a site disappear by deleting it from your New Tab page.') }}
+      <img src="{{ static('img/firefox/tiles/delete-site.png') }}" alt="{{ _('Screenshot') }}">
+    </li>
+  </ol>
+  <p>
+    <a href="https://support.mozilla.org/kb/new-tab-page-show-hide-and-customize-top-sites">{{ _('Learn more about customizing New Tab Page') }}</a>
+  </p>
 </section>
 
-<section>
-  <h3 data-accordion-role="tab">{{ _('What data is being collected?') }}</h3>
-  <div data-accordion-role="tabpanel">
-    <p>
-      {% if LANG in ['en-US'] %}
-        Mozilla collects Tiles related data such as number of clicks, impressions and Tile specific
-        data (e.g. position and size of grid) to help Mozilla determine how frequently the Tile has
-        been seen or interacted with, as well as your IP address (collected by Firefox, quickly
-        translated into a region code and then deleted).
-      {% else %}
-      {% trans %}
-        Aggregate data from the New Tab page includes impressions, clicks and tile placement.
-      {% endtrans %}
-      {% endif %}
-    </p>
-  </div>
+<section class="tls-discover">
+  <h2>{{ _('Discover new sites and content on the Web') }}</h2>
+  <p>
+    {{ _('If enabled, Firefox displays website links as thumbnails (miniaturized images) or logos on your New Tab page.') }}
+    {{ _('These are called Tiles.') }}
```

Comment:
Not sure if it was like this from the beginning, but we went a step too far in splitting sentences. Can we have one single sentence: `If enabled, Firefox displays website links as thumbnails (miniaturized images) or logos on your New Tab page. These are called Tiles.`?